kadok.go

```diff
@@ -1,5 +1,24 @@
+// Kadok is a bot developed for the Discord Guild "Les petits pedestres". It aims to provide fun and useful functionnalities for the Guilde Members.
+//
+// The code of Kadok is open source, you can use it and modify it as long as you follow the GNU_v3 licence terms.
+//
+// The current available features are:
+//
+// 1. Play ping pong
+//
+// 2. Quote characters from Kaamelott universe (and even more!)
+//
+// 3. Manage features' permission based on Discord roles
+//
+// This documentation is mainly to help developers to understand how the inside of Kadok works
+// To find more on how to start and configure the bot, please visit the wiki page: https://github.com/terag/kadok/wiki
+//
+// Main package holds the global structure of bot and is also the only package importing the SDK DiscordGo.
+// Other packages are responsible to deliver domain specific features such as security features or characters features.
 package main
 
+// BUG(Victor) : As of today, the bot is mono-guild only due to the internal structure of the permissions. If you try to run it on multiple guilds, it will work, however you may have wrong permissions granted when calling the bot
```

Comment:
please do not include your own name in the code, you are already the author of the commit adding that

Besides, please open an issue since we know we tend to never solve those kind of issue if it is only referenced within the code

Reply:
The comment is moved to the documentation in the configuration part. An issue is open #25 .